integration/fuse/bin/alluxio-fuse

```diff
@@ -19,13 +19,13 @@ umount \tunmount an Alluxio path from local file system
 stat \tshow status of Alluxio mount points"
 
 readonly MOUNT_USAGE="
-Mounts a path in Alluxio namespace (defaults to "/") to mount point on local file system.
-When no argument is given, list the current mount point
```

Comment:
@liugs0213 @maobaolong @Nizifan
when no argument is given, will not list the current mount point, but use the default `alluxio.fuse.mount.point` and `alluxio.fuse.mount.alluxio.path` to mount.